src/layers/MapboxStyleLayer/MapboxStyleLayer.js

```diff
@@ -40,6 +40,7 @@ const applyLayoutVisibility = (mbMap, visible, filterFunc) => {
 class MapboxStyleLayer extends Layer {
   constructor(options = {}) {
     super(options);
+    this.isMapboxStyleLayer = true;
```

Comment:
use duckTyping because instanceof will not work with lazy loading of intern topics

Reply:
Why don't  you specify hideInLegend prop for the layer you want to hide in legend ?

Reply:
I thought we would never want mapboxStyle Layer to appear in the layerTree.
I can add hideInLegend to all of them if it makes more sense.

Reply:
There is a lot of MapboxStyle Layer in the layer tree, see netzkarte topic

Reply:
and eisenbahninfrastruktur

Reply:
That's why I wanted to hide them all by default :smiley:

Reply:
This is fixed, for infofpw, we will fix it on gitlab